doc/doxygen/Makefile

```diff
@@ -147,13 +147,6 @@ clean-tooling:
 todo:
 	@grep -Eor "@needsref [a-zA-Z._-]+" ../ | sort | uniq | sed 's|..//||' | column -s ':' -t
 
-##
-## File targets
-##
-
-src/styles.css: src/styles.scss $(SASS)
```

Comment:
[fix] We can get rid of the `sass` dependency from `package.json` (and `package-lock.json`) now.

Reply:
Addressed 389b813f425409ef2f0cd7c75adebad00d8d5119